package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@funboxteam/diamonds",
-  "version": "6.4.1",
+  "version": "6.4.2",
   "description": "A shiny pile of typed JS helpers for everyday use",
   "scripts": {
     "lint": "eslint --cache -c .eslintrc.js --ext .ts lib",
     "prebuild": "npm run lint && rm -rf dist",
     "build-cjs": "tsc --outDir ./dist/cjs --module CommonJS",
     "build-esm-and-types": "tsc --outDir ./dist/esm --module ES6 --declaration true --declarationDir ./dist/types",
     "build": "npm run build-esm-and-types && npm run build-cjs",
-    "prepublishOnly": "npm run build"
+    "prepack": "npm run build"
```

Comment:
> Runs BEFORE a tarball is packed (on `npm pack`, `npm publish`, and when installing a git dependencies).

I think this is really helpful when testing locally and building a package with `npm pack`.

Reply:
Sounds reasonable, thanks!